rust/src/mask/config/mod.rs

```diff
@@ -129,29 +128,30 @@ pub enum ModelType {
 }
 
 impl ModelType {
+    /// Gets the number of models to be aggregated at most for this model type.
```

Comment:
I'd suggest

> Get the maximum number of models that can be aggregated for this model type

Sidenote: I've used the imperative form in the parts I've documented. I don't mind using the third person but we should agree on something to be consistent.

Reply:
changed the wording.

Reply:
i agree that we should have consistent styles, i have no strong preference for imperative or non-imperative style, i guess we should consistify this across the code base once the contributions to the documentations are all merged.